recipe_scrapers/__init__.py

```diff
@@ -459,26 +459,37 @@
 }
 
 
-def scrape_me(url_path: str, **options: dict[str, Any]) -> AbstractScraper:
+def scrape_me(
+    url_path: str,
+    proxies: Optional[Dict[str, str]] = None,
+    timeout: Optional[Union[float, Tuple[float, float], Tuple[float, None]]] = None,
+    wild_mode: Optional[bool] = None,
+    html: Union[str, bytes, None] = None,
+) -> AbstractScraper:
     host_name = get_host_name(url_path)
 
     try:
         scraper = SCRAPERS[host_name]
     except KeyError:
-        if not options.get("wild_mode", False):
+        if not wild_mode:
             raise WebsiteNotImplementedError(host_name)
         else:
-            options.pop("wild_mode")
-            wild_scraper = SchemaScraperFactory.generate(url_path, **options)
+            wild_scraper = SchemaScraperFactory.generate(
+                url_path, proxies, timeout, wild_mode, html
+            )
             if not wild_scraper.schema.data:
                 raise NoSchemaFoundInWildMode(url_path)
             return wild_scraper
 
-    return scraper(url_path, **options)
+    return scraper(url_path, proxies, timeout, wild_mode, html)
 
 
 def scrape_html(
-    html: str, org_url: Optional[str] = None, **options: dict[str, Any]
+    html: str | bytes,
```

Comment:
Note: I think the fact that the `html` argument can be either a `str` or `bytes` at the moment is probably a kind of interface bug or deficiency.  Was there a specific `mypy` warning that raised this, and if so, could we track down where the relevant code that causes it?

Reply:
I think requests can return `bytes` and the scrapers already handle `str | bytes` https://github.com/sbdchd/recipe-scrapers/blob/c954e1a087e96d5f0c4350ee2f498a366f56056e/recipe_scrapers/_abstract.py#LL33

Reply:
more specifically, I'm using `scrape_html` in a project and it passes in `bytes` from requests successfully

Reply:
Cool, ok.  I've no strong opinions on it yet, but I feel like perhaps we should drop support for `bytes` in the `scrape_html` method at some point.

It could be worth looking at some of the decoding that `requests` (and other Python HTTP libraries) can perform - if I remember correctly, it's the difference between `response.text` and `response.content` in the `requests` library.

cc @bfcarpio, since this could be input (no pun intended) for some of the networking code / interface changes (re: #658)

Reply:
yeah https://github.com/sbdchd/recipe-scrapers/blob/c954e1a087e96d5f0c4350ee2f498a366f56056e/recipe_scrapers/_abstract.py#LL46 returns `bytes`